tests/unit/helpers/dateTimeHelperTests.ts

```diff
@@ -0,0 +1,16 @@
+import 'mocha'
+import { calcuateTimeDifferenceMilliseconds } from '../../../src/helpers/dateTimeHelper'
+import { assert } from 'chai'
+
+describe('dateTimeHelper', () => {
+
+  describe('calcuateTimeDifferenceMilliseconds', () => {
```

Comment:
Just following from my previous comment
```suggestion
  describe('calculateTimeDifferenceInMilliseconds', () => {
```